package.json

```diff
@@ -353,6 +353,7 @@
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
     "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#baf106d20d971f711ab834efc9730fac5df775db",
+    "web-components": "https://github.com/department-of-veterans-affairs/component-library.git#wc-v0.2.0",
```

Comment:
The dependency is added using a git url with a tagged release. We aren't publishing this to npm right now since it is in the same repo as our `component-library` package, and having two packages published from parallel branches in the same repo seems like an even more awkward situation.